week4/Q1.py

```diff
@@ -0,0 +1,104 @@
+def _rename_islands(islands_map, old_name, new_name):
+	'''
+	Changes names of the tiles.
+
+	Args:
+		islands_map: two-dimensional list
+		old_name: int
+		new_name: int
+
+	Returns:
+		two-dimensional list
+	'''
+	for i, row in enumerate(islands_map):
+		for j, tile in enumerate(row):
+			if tile == old_name:
+				islands_map[i][j] = new_name
+	return islands_map
+
+
+def _name_island(islands_map, row, column, islands, island_counter, upper_tile, left_tile):
+	'''
+	Gives name (which island) to the new tile, renames old tiles if needed (if islands which were seen as separate actually happen to be connected).
```

Comment:
Generally, keep your line at most 80 characters wide, so that when reading them, one does not have to scroll left and right. 80 is arbitrary, but a common limit.